src/components/ModelDialog/components/ModalHeader/ModalHeader.tsx

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam, Vereniging van Nederlandse Gemeenten
+import { Close } from '@amsterdam/asc-assets'
+import { Button } from '@amsterdam/asc-ui'
+import { StyledHeading, Header } from './styled'
+
+export type Props = {
+  title: string
+  onClose?: () => void
```

Comment:
OnClose is mandatory in the component above, so no need to make it optional here.

Reply:
Removed optional